project/profiles/capacity4more/modules/c4m/content/c4m_content_wiki_page/c4m_content_wiki_page.features.field_base.inc

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @file
+ * c4m_content_wiki_page.features.field_base.inc
+ */
+
+/**
+ * Implements hook_field_default_field_bases().
+ */
+function c4m_content_wiki_page_field_default_field_bases() {
+  $field_bases = array();
+
+  // Exported field_base: 'field_c4m_editing_open'.
+  $field_bases['field_c4m_editing_open'] = array(
```

Comment:
Sounds like the "classic" job for `{node_access}` - which allows you to have arbitrary `realm`s and `gid`s, so you can have any logic written there.

See for example [OG access](https://github.com/Gizra/og/blob/c2147b5b6cac9876c30bf35ef832ea73efb45392/og_access/og_access.module#L133-L144) - you can write an many records with differenet realm and gids as you want, and as along as a user will match one of them in `hook_node_grant` (see [example](https://github.com/Gizra/og/blob/c2147b5b6cac9876c30bf35ef832ea73efb45392/og_access/og_access.module#L53-L60)) they will have access to the operation